ios/RIBs/Classes/ViewableRouter.swift

```diff
@@ -26,6 +26,13 @@ public protocol ViewableRouting: Routing {
 
     /// The base view controllable associated with this `Router`.
     var viewControllable: ViewControllable { get }
+
+    /// this block will be used as prior in case if it's not nil
+    var presentationTransition: ((_ routing:ViewableRouting,_ completion:(() -> Void)) -> Void)? { get }
```

Comment:
Have you run the linter? I suppose there will be warnings about spaces.

Reply:
@kradenkov why? looks okay

Reply:
Colon Violation: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. (colon)
Comma Spacing Violation: There should be no space before and one after any comma. (comma)

Reply:
fixed

Reply:
aaaaa

Reply:
you are totally right